progress-indicator.ts

```diff
@@ -26,5 +32,18 @@ export class ProgressIndicator implements IProgressIndicator {
 
 		return tempPromise;
 	}
+
+	public getSpinner(message: string): ISpinner {
+		if (isInteractive()) {
+			return new clui.Spinner(message);
+		} else {
+			let msg = message;
+			return {
+				start: () => this.$logger.info(msg),
```

Comment:
For cloud builds this will print for example `Installing tns-android`  and we are going to loose the broken "progress indicator" which we currently have. Without it our users may wonder is the build slow or is there some other problem (network connectivity issue on their side, or some bug on our side).
Do you think it will be better to have some kind of progress indicator instead of just one message?

Reply:
I think we shouldn't rely on the cloud build logs in order to show progress indication on the client side. IMO this should be handled on the client side in any other meaningful way.
@tsvetie what do you think?